### PR TITLE
Setup server. Finished api GET endpoint to get summary of reviews for a particular hostel with provided hostelID

### PR DESCRIPTION
**Pull request by datonedoe**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  .then(() => console.log('Connected to database'))
+  .catch(err => console.log('Failed to connect to database:', err));
+
+// Body parser middleware
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.get('/api/reviews/full/:hostelId', (req, res) => {
+  // console.log('hostelId', req.params.hostelId);
+  const { hostelId } = req.params;
```
Comment (smrutherford) [inline server/app.js:24]: Does this work? Should it be req.params.hostelId? (same comment for second get request)

File: server/app.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  Hostel.findById(hostelId)
+    .populate({
+      path: 'reviews',
+      select: ['text', 'user', 'rate', 'created_at', 'propertyResponse', '-_id'],
+      populate: { path: 'user', select: 'username' },
+    })
+    // .populate('reviews')
+    .then(data => res.json(data))
+    .catch(err => console.log('ERROR', err));
+  // res.send('hello');
```
Comment (smrutherford) [inline server/app.js:34]: Don't forget to update to send data back to client

**Review by smrutherford — APPROVED** [review]
Looks good! Don't forget to delete your commented working lines in server/app.js.

File: server/app.js (hunk 1 above)
Comment (datonedoe) [reply server/app.js:24]: @smrutherford yea it should work, I normally do as what you suggested but eslint made us deconstruct a key out of an object if you want a variable name same as the key.

**Pull request merged**